src/app/components/get-this-title/get-this-title.html

```diff
@@ -99,7 +99,7 @@
                     </a>
                 </div>
                 <div if="model.cheggLink" class="option">
-                    <a href="{model.cheggLink}"
+                    <a href="{model.cheggLink}" data-track="Chegg"
```

Comment:
Note that these are the values I am expecting: https://github.com/openstax/openstax-cms/blob/master/salesforce/models.py#L248-L256